src/main/java/uk/gov/hmcts/cmc/claimstore/services/notifications/DefendantResponseNotificationService.java

```diff
@@ -143,11 +159,14 @@ public void logNotificationFailure(
         parameters.put(DEFENDANT_NAME, claim.getClaimData().getDefendant().getName());
         parameters.put(FRONTEND_BASE_URL, notificationsProperties.getFrontendBaseUrl());
         parameters.put(CLAIM_REFERENCE_NUMBER, claim.getReferenceNumber());
+
         parameters.put(MEDIATION_DECISION_DEADLINE, formatDate(decisionDeadline));
+        parameters.put(MEDIATION_RESPONSE_DEADLINE, Formatting.formatDate(decisionDeadline));
```

Comment:
It looks like this is duplicate. A `MEDIATION_DECISION_DEADLINE` has the same value and sounds same.